src/Microsoft.Extensions.Configuration.AzureAppConfiguration/JsonConfigurationParser.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal static class JsonConfigurationParser
+    {
+        public static bool IsJsonContentType(string contentType)
+        {
+            try
+            {
+                string acceptedMainType = "application";
+                string acceptedSubType = "json";
+
+                ContentType ct = new ContentType(contentType.Trim().ToLower());
+                var type = ct.MediaType;
+                if (Equals(type, FeatureManagementConstants.ContentType)
```

Comment:
>Equals [](start = 20, length = 6)

How about using `==` in place of `Equals`?

Reply:
Is == better than Equals in this case? I found that there is no difference when comparing strings and == actually calls Equals function: [https://docs.microsoft.com/en-us/dotnet/api/system.string.op_equality?view=netcore-3.1#remarks](https://docs.microsoft.com/en-us/dotnet/api/system.string.op_equality?view=netcore-3.1#remarks) 

---
In reply to: [453142883](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r453142883) [](ancestors = 453142883)

Reply:
Both of these are value comparisons and can be used interchangeably. We currently use `==` for string comparisons in this solution, so using it would help maintain consistency.

---
In reply to: [453940673](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r453940673) [](ancestors = 453940673,453142883)

Reply:
Done

---
In reply to: [454041425](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r454041425) [](ancestors = 454041425,453940673,453142883)